python/example_robot_data/robots_loader.py

```diff
@@ -88,12 +88,8 @@ def loadANYmal(withArm=None):
                         free_flyer=True)
 
 
-def loadTalosArm():
-    return robot_loader('talos_data', "talos_left_arm.urdf", "talos.srdf")
-
-
-def loadTalos(legs=False):
-    URDF_FILENAME = "talos_reduced.urdf"
+def loadTalos(legs=False, arm=False):
+    URDF_FILENAME = "talos_left_arm.urdf" if arm else "talos_reduced.urdf"
     SRDF_FILENAME = "talos.srdf"
 
     robot = robot_loader('talos_data', URDF_FILENAME, SRDF_FILENAME, free_flyer=True)
```

Comment:
For the arm case, we should define `free_flyer` as `False`

Reply:
Thanks for your review. This is fixed, but your remark raise the following points:
1. According to `TalosArmTest`, this arm is not supposed to have a free flyer, but according to `TalosArmFloatingTest`, this arm is supposed to have a free flyer.
2. Tests are not testing anything. if I change the RobotTestCase.NQ for one or the other of those tests, the test are still passing. I was going to refactor those anyway.

Reply:
I would remove the `TalosArmFloatingTest`. The reasons are
 - it is contra-intuitive to have a "floating-base" arm
 - this unit-test doesn't bring anything new

I agree the "tests" are not that good. Nice that you will plan to improve them :)